pwiz_tools/Skyline/SettingsUI/BuildLibraryDlg.cs

```diff
@@ -36,6 +37,13 @@
 using pwiz.Skyline.ToolsUI;
 using pwiz.Skyline.Util;
 using pwiz.Skyline.Util.Extensions;
+using pwiz.Skyline.Model.Tools;
+using System.Runtime.CompilerServices;
+using File = System.IO.File;
+using Settings = pwiz.Skyline.Properties.Settings;
+
+
+[assembly: InternalsVisibleTo("TestFunctional")]

```

Comment:
These "InternalsVisibleTo" things should probably go in "Program.cs" because we already have one of them there.
Normally, we would put them in "AssemblyInfo.cs", but because "AssemblyInfo.cs" is generated by the bjam build, "Program.cs" is the next best place to put these.